com.freerdp.FreeRDP.json

```diff
@@ -0,0 +1,177 @@
+    ],
+    "add-build-extensions": {
+      "org.freedesktop.Platform.ffmpeg-full": {
+        "directory": "lib/ffmpeg",
+        "version": "22.08",
+        "add-ld-path": "."
+      },
+		"org.freedesktop.Platform.openh264": {
+        "directory": "lib/openh264",
+        "version": "2.3.0",
+        "add-ld-path": "."
+      }
+    },
+    "modules": [
+    "shared-modules/libusb/libusb.json",
+    {
+        /**
+         * libfreerdp use xprop to try to detect keyboard layout
+         */
+        "name": "xprop",
+        "sources": [
+        {
+            "type": "archive",
+            "url": "https://xorg.freedesktop.org/releases/individual/app/xprop-1.2.5.tar.bz2",
+            "sha256": "9b92ed0316bf2486121d8bac88bd1878f16b43bd335f18009b1f941f1eca93a1"
+        }
+        ],
+				"cleanup": [
+					"/share/doc",
+					"/share/man"
+				]
+    },
+    {
+        "name": "pcsc",
+        "config-opts": [
+            "--disable-libsystemd",
+            "--enable-pic",
+            "--disable-libusb",
+            "--enable-shared",
+            "--with-systemdsystemunitdir=/app/lib/systemd/"
+        ],
+        "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/LudovicRousseau/PCSC.git",
+                "tag": "1.9.9",
+                "commit": "15c16c7796607b1c8a2ce253d3f536918ab26b4a"
+            }
+        ],
+				"cleanup": [
+					"/share/doc",
+					"/share/man"
+				]
+    }, 
+		{
+			"name": "openssl111",
+			"buildsystem": "simple", 
+			"build-commands": [
+				"./config --prefix=/app",
+				"make -j $FLATPAK_BUILDER_N_JOBS",
+				"make install"
+			],
+			"cleanup": [
+				"/share/doc",
+				"/share/man",
+				"/bin"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.openssl.org/source/openssl-1.1.1q.tar.gz",
+					"sha256": "d7939ce614029cdff0b6c20f0e2e5703158a489a72b2507b8bd51bf8c8fd10ca"
+				}
+			]
+		},
+    {
+        "name": "freerdp",
+        "buildsystem": "cmake-ninja",
+        "cleanup": [
+        ],
+        "config-opts": [
+					"-DCMAKE_VERBOSE_MAKEFILE=ON", 
+					"-DCMAKE_BUILD_TYPE:STRING=Release",
+					"-DCMAKE_INSTALL_LIBDIR:PATH=lib",
+					"-DCHANNEL_TSMF:BOOL=OFF",
+					"-DCHANNEL_URBDRC:BOOL=ON",
+					"-DBUILD_TESTING:BOOL=OFF",
+					"-DWITH_MANPAGES:BOOL=OFF",
+					"-DWITH_GSSAPI:BOOL=OFF",
+					"-DWITH_PCSC:BOOL=ON",
+					"-DWITH_PKCS11:BOOL=ON",
+					"-DWITH_SWSCALE:BOOL=ON",
+					"-DWITH_SERVER:BOOL=ON",
+					"-DWITH_SAMPLE:BOOL=ON",
+					"-DWITH_CUPS:BOOL=ON",
+					"-DWITH_FFMPEG:BOOL=ON",
+					"-DWITH_DSP_FFMPEG:BOOL=ON",
+					"-DWITH_OSS:BOOL=OFF",
+					"-DWITH_PULSE:BOOL=ON"
+        ],
+        "sources": [
+        {
+            "type": "git",
+            "url": "https://github.com/FreeRDP/FreeRDP.git",
+            "tag": "2.8.0",
+            "commit": "e3fc97feb512053189e276b2ca79762990bb8c4c"
+        }
+        ]
+    },
+    {
+        "name": "freerdp-icons",
```

Comment:
there is no reason to have this in a separate module since it's the same source as above.

Reply:
@hfiguiere ok, can I use different build systems on the same module? (sorry, not my first flatpak but not done too many others)

Reply:
ok, found the `post-install` array